src/entities/Proposal/model.ts

```diff
@@ -358,13 +350,13 @@ export default class ProposalModel extends Model<ProposalAttributes> {
 
     const timeFrame = this.parseTimeframe(filter.timeFrame)
     const timeFrameKey = filter.timeFrameKey || 'created_at'
-    const orderDirection = order || 'DESC'
+    const orderDirection = !order || order === 'RELEVANCE' ? SortingOrder.DESC : order
```

Comment:
What does `!order || order === 'RELEVANCE'` means? Could you move that to a variable? You can reuse it here and in line 359.